lib/vdsm/host/caps.py

```diff
@@ -175,6 +175,7 @@ def get():
     caps['measure_active'] = True
     caps['mailbox_events'] = config.getboolean("mailbox", "events_enable")
     caps['zerocopy_migrations'] = hasattr(libvirt, 'VIR_MIGRATE_ZEROCOPY')
+    caps['qemu_image_info_bitmaps'] = True
```

Comment:
we'll need to add this and use it on the engine patch as well

Reply:
Indeed! I'll try to fix the engine patch next week